File: app/src/main/java/com/druger/aboutwork/model/CompanyResponse.java
package com.druger.aboutwork.model;

import java.util.List;

/**
 * Created by druger on 30.07.2016.
 */
public class CompanyResponse {

    public static final int PER_PAGE = 20;
    private int page;
    private int pages;
    private int found;
    private List<Company> items;

    public int getPage() {
        return page;
    }

    public void setPage(int page) {
        this.page = page;
    }

    public int getPages() {
        return pages;
    }

    public void setPages(int pages) {
        this.pages = pages;
    }

    public int getFound() {
        return found;
    }

    public void setFound(int found) {
        this.found = found;
    }

    public List<Company> getItems() {
        return items;
    }

    public void setItems(List<Company> items) {
        this.items = items;
    }
}

File: app/src/main/java/com/druger/aboutwork/rest/models/CompanyResponse.kt
package com.druger.aboutwork.rest.models

import com.druger.aboutwork.model.Company

/**
 * Created by druger on 30.07.2016.
 */
class CompanyResponse {
    var page: Int = 0
    var pages: Int = 0
    var found: Int = 0
    var items: List<Company>? = null

    companion object {

        val PER_PAGE = 20
    }
}
